api/v1/controllers/samples.js

```diff
@@ -53,7 +53,9 @@ module.exports = {
   findSamples(req, res, next) {
```

Comment:
So theoretically we should be able to run *all* the api sample tests with the sampleStore feature flag enabled and everything should work?